### PR TITLE
4.0 commit

### PR DESCRIPTION
**Pull request by GuildedThorn**

### CODE REVIEW

File: src/CupidonSauce173/PigNotify/UI.php [hunk 1]
```diff
@@ -7,8 +7,8 @@
 
 use CupidonSauce173\PigNotify\lib\FormAPI;
 use CupidonSauce173\PigNotify\Object\Notification;
-use pocketmine\Player;
 
+use pocketmine\player\Player;
```
Comment (CupidonSauce173) [inline src/CupidonSauce173/PigNotify/UI.php:11]: Why did it move ?

File: .idea/discord.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (CupidonSauce173) [inline .idea/discord.xml:1]: What is this file?

**Review by CupidonSauce173 — CHANGES_REQUESTED** [review]
File .idea/discord.xml : why did you add this ?

File: .idea/discord.xml (hunk 2 above)
Comment (opticalPvPX) [reply .idea/discord.xml:1]: its for phpstorm discord RPC can you remove it from the pr please?

**Pull request merged**